debian/postinst

```diff
@@ -3,9 +3,11 @@
 set -e
 
 ESM_APT_GPG_KEY="/etc/apt/trusted.gpg.d/ubuntu-esm-v2-keyring.gpg"
+ESM_APT_GPG_KEY_V1="/etc/apt/trusted.gpg.d/ubuntu-esm-keyring.gpg"
```

Comment:
bikeshedding, but can we call it something else? like old?

Reply:
funny I had OLD but changed to V1 in case we end up retiring V2 too.
I don't mind switching it.